git-version-control/gitignore.md

```diff
@@ -126,8 +118,9 @@ You should no longer see *.my-secret-keys* in your untracked files, because git
 
 ### Check for Understanding
 
-- Small group discussion:
+Form small groups and discuss:
 
   - What sorts of files should you add to your .gitignore?
-
   - Should you commit your .gitignore file?
+  - Should you add folders to your .gitignore file?
```

Comment:
What answer are you expecting for this?  I'm not sure we need this line.

Reply:
Yeah yeah, I'm not sure what I was thinking with this. I will remove.